code/modules/projectiles/projectile/bullets.dm

```diff
@@ -181,18 +181,18 @@
 /* "Rifle" rounds */
 
 /obj/item/projectile/bullet/rifle
-	armor_penetration = 20
 	penetrating = 1
 
 /obj/item/projectile/bullet/rifle/a762
-	damage = 25
+	damage = 35
 
 /obj/item/projectile/bullet/rifle/a556
 	damage = 30
+	armor_penetration = 20
 
 /obj/item/projectile/bullet/rifle/a556/ap
-	damage = 25
-	armor_penetration = 25
+	damage = 30
+	armor_penetration = 45
```

Comment:
this is an excessive amount of AP

Reply:
If I understand the code correctly, previously ALL rifle bullets had a base AP of 20. I removed that- and the AP 5.56 is the same as it was before. Correct me if I'm wrong.

Reply:
it isn't additive, you're setting a value to something here - what it was before was 25, not 45

Reply:
Alright, I'll fix this then